### ISSUE TITLE
No commands defined in the "m2-migration" namespace

### ISSUE THREAD

**Issue by drinkingsouls** [peterjaap/magerun-addons#65]
Hi,
This addon could be a lifesaver if I could get it working as I have some missing delta information.

In my root directory I have mangerun and the addons installed:
root/n98-magerun2.phar (main extension)
root/.n98-magerun (addons, all subdirectories populated)
My magento 2 is in root/public_html

If I run `n98-magerun.phar m2-migration:delta-update-changelog` in public_html I get the error 
`There are no commands defined in the "m2-migration" `namespace.``

If I run it in root/.n98-magerun/modules I get:
` Magento folder could not be detected`

I can run n98-magerun2.phar in public_html fine but it does not have the extra commands.

Where should the extension be installed? I read the Magerun docs which recommended root directory.
Thank you.

- Just to clarify, this is supposed to be installed on Magento 2 or Magento 1?

**Comment by peterjaap** [comment]
@drinkingsouls this is for Magento 1.

It should be installed in `~/.n98-magerun/modules/`. The directory name does not matter as long as it is unique. Also see https://github.com/peterjaap/magerun-addons#installation.

**Comment by drinkingsouls** [comment]
Thank you, I managed to get it working. I was assuming it was for Magento 2.
For reference and who it may help.
I installed Magerun on Magento 1 in the root directory.
I installed this module in ~/.n98-magerun/modules/ following the install instructions.
Once installed, run command in the root directory with your Magento 2 database credentials:
`n98-magerun.phar m2-migration:delta-update-changelog --host="localhost" --dbname="mage2_dbname" --username="mage2_dbuser" --password="mage2dbpassword" --prefix="mage2_"`
